package.json

```diff
@@ -27,25 +27,25 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "~9.1.0",
-    "@angular/cdk": "^9.2.0",
-    "@angular/common": "~9.1.0",
-    "@angular/compiler": "~9.1.0",
-    "@angular/core": "~9.1.0",
-    "@angular/elements": "^9.1.0",
-    "@angular/flex-layout": "^9.0.0-beta.29",
-    "@angular/forms": "~9.1.0",
-    "@angular/material": "^9.2.0",
-    "@angular/platform-browser": "~9.1.0",
-    "@angular/platform-browser-dynamic": "~9.1.0",
-    "@angular/router": "~9.1.0",
+    "@angular/animations": "^10.1.0",
+    "@angular/cdk": "^10.2.0",
+    "@angular/common": "^10.1.0",
+    "@angular/compiler": "^10.1.0",
+    "@angular/core": "^10.1.0",
+    "@angular/elements": "^10.1.0",
+    "@angular/flex-layout": "^10.0.0-beta.32",
+    "@angular/forms": "^10.1.0",
+    "@angular/material": "^10.2.0",
+    "@angular/platform-browser": "^10.1.0",
+    "@angular/platform-browser-dynamic": "^10.1.0",
+    "@angular/router": "^10.1.0",
     "@aspnet/signalr": "^1.1.4",
-    "@lcu/common": "^1.40.182-integration",
-    "@lowcodeunit-devkit/ng-lcu": "^1.5.26",
-    "@lowcodeunit/lazy-element": "0.0.13",
-    "@swimlane/ngx-charts": "^12.1.0",
-    "@webcomponents/custom-elements": "^1.4.1",
-    "@webcomponents/webcomponentsjs": "^2.4.3",
+    "@lcu/common": "^1.42.341",
+    "@lowcodeunit-devkit/ng-lcu": "^1.12.71",
+    "@lowcodeunit/lazy-element": "^1.1.22-8193-angular-10",
```

Comment:
For integration and master we need to remove feature branch versions for dependencies. I didn't notice if this was in the integration push.